.github/workflows/release-upload-helm-charts.yaml

```diff
@@ -0,0 +1,38 @@
+name: Upload Helm Charts as Release Artifacts
```

Comment:
We should probably rename this script since it's not just doing the helm charts anymore.

Reply:
Done.